packages/atlas/atlas.config.yml

```diff
@@ -511,7 +485,10 @@ legal:
 
     ## 2. Changes to Terms
 
-    This Agreement may be modified or updated at the sole discretion of Company without notice. Your continued use of our Software is confirmation of your acceptance of the latest Agreement.
+    This Agreement may be modified or updated at the sole discretion of Company in a unilateral way, without prior notice.  In case of significant change of terms the Company takes the obligation to take reasonable efforts to notify its customers about the change of the Terms. 
+    This may take a non-personalised form, such as a banner in the app for all users, post on company's Discord server channel or Company's social media accounts. It is your obligation to follow the company's communication channels to stay informed about the changes. 
```

Comment:
```suggestion
    This may take a non-personalised form, such as a banner in the app for all users, post on Company's Discord server channel or Company's social media accounts. It is your obligation to follow the Company's communication channels to stay informed about the changes. 
```